library/core/src/array/mod.rs

```diff
@@ -787,7 +787,7 @@ where
 /// `next` at most `N` times, the iterator can still be used afterwards to
 /// retrieve the remaining items.
 ///
-/// If `iter.next()` panicks, all items already yielded by the iterator are
+/// If `iter.next()` panics, all items already yielded by the iterator are
```

Comment:
I don't know which of these is technically more correct, but I know that I find the version before this PR to be easier to read/understand. I read "panics" as a plural noun by default ("one panic, two panics") and need to restart reading the sentence when that isn't what it means (such as this case), whereas "panicks" is unambiguously third-person singular simple present ("it panicked, it was panicking, it panicks often"). Since Rust commonly uses both noun and verb meaning of panic, I think it would be fine for us to use "panicks" for the verb meaning.

Reply:
@dtolnay As far as I can tell, we more-or-less consistently use "panics" as a verb almost everywhere (since most of the time we don't have to deal with more than one panic at once). "panicks" would be inconsistent (in addition to not being correct in English).

Reply:
@dtolnay @joshtriplett 
ok, sounds well.